docs/unmanaged_configuration.md

```diff
@@ -25,3 +25,50 @@ oc set env dc/fluentd MERGE_JSON_LOG=true
 * Possible log loss due to Elasticsearch rejecting documents due to inconsistent type mappings
 * Potential buffer storage leak caused by rejected message cycling
 * Overwrite of data for field with same names
+
+## ElasticSearch
+### openshift_logging_es_allow_external
+If openshift_logging_es_allow_external is set to True, Elasticsearch was exposed as a Route.
+The following steps will achieve the goal.
+* Create a route for ElasticSearch.
+```
+$ oc create -f - << EOF
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: elasticsearch
+  namespace: openshift-logging
+spec:
+  host: elasticsearch.openshift-logging.svc.cluster.local
+  to:
+    kind: Service
+    name: elasticsearch
+  tls:
+    termination: passthrough
+EOF
+```
+* Get the routerCanonicalHostname
+```
+$ myRouterCanonicalHostname=$( oc get routes ingress[*].routerCanonicalHostname}' )
+```
+* Get the IP address of the ElasticSearch pod
+```
+for i in `nslookup openshift-logging.$myRouterCanonicalHostname | egrep Address | awk '{print $2}' | grep -v \#`; do ESIPADDR=$i; done
+```
+* Get the ElasticSearch FQDN
+```
+$ ESFQDN=$( oc get routes elasticsearch -o jsonpath='{.spec.host}' )
+```
+* Add the ElasticSearch IP address and FQDN into /etc/hosts on the host the ElasticSearch is to be accessed.
+```
+# echo $ESIPADDR $ESFQDN >> /etc/hosts
+```
+* Get admin key, cert and CA cert from ElasticSearch secret
+```
+for i in admin-key admin-ca admin-cert; do oc exec $ESPOD -- cat /etc/openshift/elasticsearch/secret/$i > $i ; done
+```
+Assume the files are located at /path/to/{admin-ca,admin-cert,admin-key} on the host.
```

Comment:
Assuming re-encrypt, these are the certs you, as an admin, provide? we should name them differently.  Do we need to provide a set of certs for re-encrypt or just the dest_ca_crt?

Reply:
I tested with just the dest_ca_crt and verified it worked.  Thus, I'm going to drop the set of certs. :)

I got another question here...  I compared with the route for kibana which has both dest_ca_crt and ca_crt sharing the same cert value.  Are there any reason to do so?  Should the ElasticSearch route follow that way - having both dest_ca_crt and ca_crt?  Or having just the dest_ca_crt is good enough?

Reply:
We should do whatever openshift-ansible did when it created the reencrypt route for Elasticsearch.  If it is reusing the certs, that's fine.  If it is creating new certs, then we will have to do that too.

Reply:
Ok.  This is the output from v3.11 elasticsearch route created with openshift_logging_es_allow_external=True.
http://pastebin.test.redhat.com/714785

I.e., there are both dest_ca_crt and ca_crt are placed.  I'd follow it.  Thanks!